Decompiler/Program.cs

```diff
@@ -596,19 +610,25 @@ private static void ParseVPK(string path)
 
             sw.Stop();
 
-            Console.WriteLine("Processed in {0}ms", sw.ElapsedMilliseconds);
+            if (!Options.Silent) Console.WriteLine("Processed in {0}ms", sw.ElapsedMilliseconds);
         }
 
         private static void DumpVPK(Package package, string type, string newType)
         {
+            if (!string.IsNullOrEmpty(Options.FilterExt) && !Options.FilterExt.Contains(type))
+            {
+                return;
+            }
+
             if (!package.Entries.ContainsKey(type))
             {
-                Console.WriteLine("There are no files of type \"{0}\".", type);
+                if (!Options.Silent) Console.WriteLine("There are no files of type \"{0}\".", type);
 
                 return;
             }
 
             var entries = package.Entries[type];
+            uint oldCrc32;            
```

Comment:
Looks like AppVeyor still uses an older version and fails the build, too.
What's the harm in keeping it VS2015-compatible?
```c#
uint oldCrc32;
if (OldPakManifest.TryGetValue(filePath, out oldCrc32) && oldCrc32 == file.CRC32)
```
Well, I can see one -  restraining from using this coding style in the future :)